### PR TITLE
Pull Request for Issue1840: Modify styling for AMToolButton

### PR DESCRIPTION
**Pull request by helfrij**
- Modified AMToolButton to have an enum 'colorState' property with values that would correspond to a specific button coloring (eg. Good, Bad, Neutral, None... etc). Added getter, setter, signal for the state. Declared this property using the Q_PROPERTY and Q_ENUMS macros; this allows for widgets to be selected by this property in .qss files.
- Created AMToolButton.qss, a file containing the basic/default style information for AMToolButtons. Added this file to stylesheets.qrc and acquamanCommon.pri.
- Modified AMDatamanAppController to apply AMToolButton.qss to the application.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @iainworkman, would you review?

File: source/ui/AMToolButton.h [hunk 1]
```diff
@@ -7,22 +7,32 @@ class AMToolButton : public QToolButton
 {
 	Q_OBJECT
 
+	Q_PROPERTY(ColorState colorState READ colorState WRITE setColorState NOTIFY colorStateChanged)
```
Comment (iainworkman) [inline source/ui/AMToolButton.h]: What is the use of the QProperty here? I know they're more general than the way we usually use them with AMDbObject, but what is it doing here?

Comment (helfrij) [inline source/ui/AMToolButton.h]: By defining the colorState as a QProperty, we can use it in a CSS-style selector, in AMToolButton.qss.

File: source/ui/BioXAS/BioXASPersistentView.cpp [hunk 2]
```diff
@@ -27,21 +27,51 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
...
     QWidget(parent)
 {
 	// Create and set main layout.
 
 	QVBoxLayout *layout = new QVBoxLayout();
-	layout->setMargin(0);
-
 	setLayout(layout);
 
+	// Testing.
```
Comment (iainworkman) [inline source/ui/BioXAS/BioXASPersistentView.cpp]: Can these tests be removed?

File: source/ui/AMToolButton.h (hunk 1 above)
Comment (iainworkman) [inline source/ui/AMToolButton.h]: Ahhh awesome!

**Comment by iainworkman** [comment]
Looks really great. I need to look more into using stylesheets like this.
Time Estimate: 10 minutes

**Comment by dretrex** [comment]
This is super cool!  Awesome work!

Time Estimate: 2 minutes

**Pull request merged**